db/seeds.rb

```diff
@@ -153,7 +153,7 @@ def format_errors(model = nil)
   },
 
   {:name              => :katello_promote_errata,
-   :description       => N_('A post-promotion summary of hosts with available errata'),
+   :description       => N_('A post-promotion summary of hosts with installable errata'),
```

Comment:
At least Foreman, seeds are only ever created not updated directly, not sure if we want to hold to the same style.
If so, I think this should have a corresponding migration.  Otherwise the `find_or_create` would need to be changed.